src/components/Comment/index.tsx

```diff
@@ -40,29 +42,39 @@ const Comment: FC<PropsType> = ({ type, data }) => {
     setIsMore((prev) => !prev);
   }, []);
 
+  const onProfileClick = () => {
+    if (writerType === "admin") {
+      return;
+    }
+    navigate(`/user/${data.writerId}`);
+  };
+
   return (
     <S.Container>
-      <S.Image />
+      <S.Image
+        src={data.thumbnailUrl}
+        emoji={data.emoji}
+        onClick={() => onProfileClick()}
+      />
       <S.ContentContainer
         color={bgColorMap.get(type)}
         border={type === "project" ? 0 : 1}
       >
         <S.NameContainer>
           <S.Name>
-            {writerSno} {writerName} {writerType === "admin" && "선생님"}
+            {writerSno} {writerName} {writerType === "admin"}
```

Comment:
```suggestion
            {writerSno} {writerName} {writerType === "admin"}
```
여기에서는 type이 선생님일때 문자열 "선생님"을 이름 뒤에 붙인다는 의미로 넣은거라 필요해요
undo 부탁

Reply:
아하! 아 따로 뒤에 붙인다는 거엿구만 오키오키

Reply:
주석 확인했으면 지울까요?

Reply:
지웁시다